packages/httpsnippet-client-api/test/index.test.js

```diff
@@ -110,16 +132,106 @@ describe('httpsnippet-client-api', function () {
 
   describe('snippets', function () {
     SNIPPETS.forEach(snippet => {
-      it(`should generate \`${snippet}\` snippet`, async function () {
-        const [har, definition] = await getSnippetDataset(snippet);
-        const expected = await fs.readFile(path.join(DATASETS_DIR, snippet, 'output.js'), 'utf-8');
+      describe(snippet, function () {
+        let consoleStub;
+
+        beforeEach(function () {
+          try {
+            // Since we're doing integration testing with these snippets against the real `api`
+            // library we should clear out the cache that it creates so our tests will run in a
+            // cleanroom environment.
+            rimraf.sync(path.join(__dirname, '../node_modules/.cache/api'));
+          } catch (err) {
+            // If we couldn't delete the `api` specs cache then it probably doesn't exist yet.
+          }
+
+          consoleStub = sinon.stub(console, 'log');
+        });
 
-        const code = new HTTPSnippet(har).convert('node', 'api', {
-          apiDefinitionUri: 'https://example.com/openapi.json',
-          apiDefinition: definition,
+        afterEach(function () {
+          consoleStub.restore();
         });
 
-        expect(`${code}\n`).to.equal(expected);
+        it('should generate the expected snippet', async function () {
+          const [har, definition] = await getSnippetDataset(snippet);
+          const expected = await fs.readFile(path.join(DATASETS_DIR, snippet, 'output.js'), 'utf-8');
+
+          const code = new HTTPSnippet(har).convert('node', 'api', {
+            apiDefinitionUri: 'https://example.com/openapi.json',
```

Comment:
Why is the path here `/openapi.json` and not `/openapi-${snippet}.json`?

Reply:
No particular reason but I'll update this so it's a bit more contextual to each snippet and more in line with how the snippet VM runner mocks them out.

Reply:
Gotcha! I'm only scanning this test but I'm surprised it passes, considering [the nock is different](https://github.com/readmeio/api/pull/422/files/bcbfac97bcc8aeef2a1e07d9c65ef4097a095b11#diff-ae672312e784c6f29f666bc87496973fde59885728afef764ac01a128cc75b56R173)

Reply:
I'm updating them to `https://api.example.com/{snippet}.json`